src/TourPortal.js

```diff
@@ -527,8 +527,8 @@ class TourPortal extends Component {
                           ? onRequestClose
                           : () => {}
                         : typeof nextStep === 'function'
-                          ? nextStep
```

Comment:
My preference but I actually like the indents.

Reply:
here is working `.prettierrc`, you mean change settings there?